package.json

```diff
@@ -17,7 +17,7 @@
     "ghpages": "ghpages -p gh-pages/",
     "lint": "semistandard -v | snazzy",
     "lint:fix": "semistandard --fix",
-    "precommit": "npm run lint",
+    "porecommit": "npm run lint",
```

Comment:
Typo

Reply:
Ooops! That should have been removed. Before I updated dependencies in package.json, npm run lint was failing at several places. This has been reverted to "precommit".